src/helper/selection-tools/move-tool.js

```diff
@@ -209,7 +209,7 @@ class MoveTool {
             if (item.data && item.data.origPos && !item.position.equals(item.data.origPos)) {
                 moved = true;
             }
-            item.data.origPos = null;
+            delete item.data.origPos;
```

Comment:
Should this move into if (item.data && item.data.origPos) ?

Reply:
Alternatively maybe the above if doesn't need to check for those two things?

Reply:
I removed the check for `item.data` since we were unconditionally accessing `item.data.origPos` anyway, and moved the `delete` inside the conditional.